direct/types.py

```diff
@@ -57,6 +59,16 @@ class TransformKey(DirectEnum):
     SAMPLING_MASK = "sampling_mask"
     ACS_MASK = "acs_mask"
     SCALING_FACTOR = "scaling_factor"
+    REFERENCE_IMAGE = "reference_image"
+    MOVING_IMAGE = "moving_image"
+    WARPED_IMAGE = "warped_image"
+    DISPLACEMENT_FIELD = "displacement_field"
+
+
+class MaskFuncMode(DirectEnum):
+    STATIC = "static"
+    DYNAMIC = "dynamic"
+    MULTISLICE = "multislice"
```

Comment:
is str, DirectEnum not required? Also, ALL_CAPS is better and I think pep8 or something will also suggest that.

If you do it as a string you can do MaskFuncMode.STATIC == "STATIC"

Reply:
DirectEnum is defined as str, like this `class DirectEnum(str, Enum):...`.

I believe the standard is CAPS = "lower" in Enums, and also this is what we've had so far. 

E.g.
```
from torchvision.transforms import functional as F
F.InterpolationMode.NEAREST
>>> <InterpolationMode.NEAREST: 'nearest'>
```